.vscode/launch.json

```diff
@@ -345,6 +345,42 @@
         "--input", "../.vscode/hyaline/merge-documentation/input-3.sqlite",
         "--output", "../.vscode/hyaline/merge-documentation/merged.db"
       ]
+    },{
+      "name": "Check PR",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "preLaunchTask": "cleanup-check-pr",
+      "cwd": "${workspaceFolder}/cli/",
+      "program": "${workspaceFolder}/cli/cmd/hyaline.go",
+      "envFile": "${workspaceFolder}/cli/.env",
+      "args":[
+        "--debug",
+        "check", "pr",
+        "--config", "../.vscode/hyaline/check-pr/hyaline.yml",
+        "--documentation", "../.vscode/hyaline/check-pr/documentation.sqlite",
+        "--pull-request", "appgardenstudios/hyaline-example/9",
+        "--issue", "appgardenstudios/hyaline-example/2",
+        "--issue", "appgardenstudios/hyaline-example/3",
+        "--output", "../.vscode/hyaline/check-pr/comment.json"
```

Comment:
Going along with the comment about making the output recommendations in both check commands, lets call the output recommendations.json like the output of check diff.